src/test/java/com/google/neighborgood/UserInfoServletTest.java

```diff
@@ -181,14 +180,14 @@ public void normalMultipleInputTest() throws IOException {
     assertEquals(3, ds.prepare(new Query("UserInfo")).countEntities(withLimit(10)));
 
     // Filter out the entity that has the userId of 1234567890
-    PreparedQuery results =
-        ds.prepare(
-            new Query("UserInfo")
-                .setFilter(new FilterPredicate("userId", FilterOperator.EQUAL, "1234567890")));
-    Entity entity = results.asSingleEntity();
-
-    // The entity can't be null
-    assertNotNull(entity);
+    Key userEntityKey = KeyFactory.createKey("UserInfo", "1234567890");
+    Entity entity = null;
+    try {
+      entity = ds.get(userEntityKey);
+    } catch (EntityNotFoundException e) {
+      // The entity can't be null
+      assertNotNull(entity);
```

Comment:
I think the assertNotNull statement is in the wrong place?

Reply:
I'll move it outside of the catch statement :)